collect_app/src/main/res/drawable/ic_red_point.xml

```diff
@@ -0,0 +1,8 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="36dp"
+        android:height="36dp"
+        android:viewportWidth="36"
+        android:viewportHeight="36">
+    <path android:fillColor="#ffff0000" android:pathData="M18,13c-2.77,0 -5,2.24 -5,5s2.24,5 5,5s5,-2.24 5,-5s-2.24,-5 -5,-5z"/>
```

Comment:
(probably fine here since the file is explicitly called `red_point` and this is pure red)

Reply:
I renamed this `map_point` because I moved the colour into `colors.xml`.